src/sources/prometheus/remote_write.rs

```diff
@@ -29,6 +29,7 @@ pub struct PrometheusRemoteWriteConfig {
     /// The address to accept connections on.
```

Comment:
```suggestion
    /// The socket address to accept connections on.
```

We had discussion about this one.

Reply:
I'm continually forgetting about this.

Reply:
I wonder if we should just do a PR to update the rest, independently of a specific source.

Reply:
I checked and it looks not too bad, I'll work on that.

Reply:
#16035